rest/specs/src/main/filtered-resources/openapi/ingestapi.yaml

```diff
@@ -0,0 +1,221 @@
+          schema:
+            $ref: '#/components/schemas/ErrorMessage'
+    Forbidden:
+        description: >
+          The client does not have permission.
+        headers:
+          'Content-Version':
+            $ref: '#/components/headers/ContentVersion'
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ErrorMessage'
+    NotImplemented:
+      description: >
+        The requested API version is not supported and therefore not implemented.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorMessage'
+    BadRequest:
+      description: >
+        The client message could not understood by the server due to invalid format or syntax.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorMessage'
+    DefaultError:
+      description: >
+        Any other possible errors not currently known.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorMessage'
+  parameters:
+    Accept-Version:
+      name: Accept-Version
+      in: header
+      description: >
+        The minimal API version that a client using this API will accept
+        responses from.
+      required: true
+      schema:
+        $ref: '#/components/schemas/Version'
+  headers:
+    ContentVersion:
+      description: >
+        The API version used by the server to produce the REST message. The server will accept
+        messages for any minor versions prior to this one.
+      required: true
+      schema:
+        $ref: '#/components/schemas/Version'
+      examples:
+        major.minor:
+          value: '1.2'
+          summary: A released minor version
+        major.minor.patch:
+          value: '1.0.3'
+          summary: A patched version
+        dev:
+          value: '1.2.0-SNAPSHOT'
+          summary: A development version
+  schemas:
+    Product:
+      description: The JSON description of the Product being ingested
+      type: object
+      properties:
+        productInfo:
+          type: object
+          properties:
+            title:
+              type: string
+              description: The title of the Product
+              example: A Picture
+            filename:
+              type: string
+              description: The name of the source file
+              example: product.jpeg
+            bytes:
+              type: integer
+              description: The size of the file payload
+              minimum: 0
+              exclusiveMinimum: true
+              example: 1024
+            mime-type:
```

Comment:
❗️ Should not use dashes but camelcase starting with lowercase letter. These will correspond to method parameters later.